### PR TITLE
Web UI button fix

### LINKED ISSUE #2162: DIY 900 TX TTGO V1 SX127x - WIFI - Buttons - Save issues
## Current Behavior

Button Actions -> Click "Save" -> no network activity to actually save buttons settings
Open browser console - 
```
scan.js:38 Uncaught TypeError: Cannot set properties of undefined (setting 'color')
    at HTMLButtonElement.submitButtonActions (scan.js:38:23)
```

`
    buttonActions[1].color = to8bit(_(`button2-color`).value)`

### PR DESCRIPTION
**Pull request by pkendall64**
Checks if the button entry is undefined before trying to store data in it.
Also fixes the feedback dialog when saving the button actions.

Fixes #2162 

## Testing

1. Use a TTGO (or other TX module with a single button or comment out the second button definition in the Radiomaster Ranger JSON file)
2. Boot into wifi mode, go to the button tab and press the save button. There should not an error in the web-console and you should see the success dialog.

### CODE REVIEW

**Comment by JyeSmith** [comment]
I dont currently have my Ranger to test 😞

File: src/html/scan.js [hunk 1]
```diff
@@ -591,18 +591,25 @@ function submitButtonActions(e) {
...
+          title: 'Success',
+          message: 'Button actions have been saved'
+        });
+      } else {
+        cuteAlert({
+          type: 'error',
+          title: 'Failed',
+          message: 'An error occurred while saving button configuration'
+        });
+      }
```
Comment (schugabe) [inline src/html/scan.js:612]: } missing

**Review by schugabe — APPROVED** [review]
With the missing } it fixes the issue

File: src/html/scan.js (hunk 1 above)
Comment (pkendall64) [reply src/html/scan.js:612]: How did I not see that in my testing 😓

Comment (schugabe) [reply src/html/scan.js:612]: I don't know how it worked during testing but it does probably not help that vs code does not like the @ @  code in the js file and is painting some lines red :sweat_smile:

**Review by CapnBry — APPROVED** [review]
* Flashed master to Ranger, verified both buttons / colors working, able to save values and persist across reboots. Do not get any confirmation popup when saving values. :( :x:
* Removed "Button 2 pin" and "Button 2 RGB Index" from hardware.html
* Button 2 now missing from webui, but can't save Button 1's stuff due to "Uncaught type error" in javscript :x:
* Flashed PR code, can save single button and single LED settings. Do get confirmation popup. ✔️ 
* Reset hardware config, can save 2 button and LED settings. Do get confirmation popup ✔️
* Also tried to trick it by removing "Button 2 pin" but leaving "Button 2 RGB Index". Does not throw any errors saving. The RGB LED still works, but obviously no way to change the value. Get another button, fool! ✔️

**Pull request merged**